packages/trader/src/Modules/Contract/Containers/contract-replay-widget.tsx

```diff
@@ -6,14 +6,16 @@ import TopWidgets from '../../SmartChart/Components/top-widgets';
 import { observer, useStore } from '@deriv/stores';
 
 export const DigitsWidget = observer(() => {
-    const { contract_replay } = useStore();
+    const { contract_replay, ui } = useStore();
+    const { is_mobile } = ui;
     const { contract_store } = contract_replay;
     const { contract_info, digits_info, display_status, is_digit_contract, is_ended } = contract_store;
 
     return (
         <Digits
             is_digit_contract={is_digit_contract}
             is_ended={is_ended}
+            is_mobile={is_mobile}
```

Comment:
Extract is_mobile in order to replace deprecated `isMobile()` in packages/trader/src/Modules/Contract/Components/Digits/digits.tsx. Also added sanity check for this case in the card